assets/app.js

```diff
@@ -0,0 +1,33 @@
+/* global hoodie */
+
+document.body.addEventListener('submit', function (event) {
```

Comment:
I don’t know to be honest, give it a try? And good idea with the comments, wanna add that?